click/_winconsole.py

```diff
@@ -209,13 +233,21 @@ def _get_text_stdin(buffer_stream):
 
 
 def _get_text_stdout(buffer_stream):
+    if PY2:
+        buffer_stream = StreamWrapper(buffer_stream)
+        sys.stdout = StreamWrapper(sys.stdout)
```

Comment:
`_compat.get_binary_stream` also calls `setmode`, might need to think more about where this patch happens.

Reply:
Yeah. Probably need to rethink where to apply the patch. See https://github.com/pallets/click/pull/830/files#r219271929.